### ISSUE TITLE
Simple question

### ISSUE THREAD

**Issue by xsr6064** [mozilla/TTS#64]
First of all, thanks a lot for sharing the great work. 

Then I have a simple question since I am pretty new to Pytorch: is it possible to add some simple comments to the parameters in the config files to help people like me to better understand the code? For example, could you let me know what the parameter 'wd' (in config.json) does? I notice that it is used to update params after the back-prog step, but not sure what the purpose of the update is. 

Thanks in advance for any clues.

**Comment by erogol** [comment]
It is actually done for the next release but for your help here is the new version. Hope it helps.

```
{
    "model_name": "TTS-dist",
    "model_description": "Distributed training.",

    "audio":{
        "audio_processor": "audio",     // to use dictate different audio processors, if available.
        // Audio processing parameters
        "num_mels": 80,         // size of the mel spec frame. 
        "num_freq": 1025,       // number of stft frequency levels. Size of the linear spectogram frame.
        "sample_rate": 22050,   // wav sample-rate. If different than the original data, it is resampled.
        "frame_length_ms": 50,  // stft window length in ms.
        "frame_shift_ms": 12.5, // stft window hop-lengh in ms.
        "preemphasis": 0.97,    // pre-emphasis to reduce spec noise and make it more structured. If 0.0, no -pre-emphasis.
        "min_level_db": -100,   // normalization range
        "ref_level_db": 20,     // reference level db, theoretically 20db is the sound of air.
        "power": 1.5,           // value to sharpen wav signals after GL algorithm.
        "griffin_lim_iters": 60,// #griffin-lim iterations. 30-60 is a good range. Larger the value, slower the generation.
        // Normalization parameters
        "signal_norm": true,    // normalize the spec values in range [0, 1]
        "symmetric_norm": false, // move normalization to range [-1, 1]
        "max_norm": 1,          // scale normalization to range [-max_norm, max_norm] or [0, max_norm]
        "clip_norm": true,      // clip normalized values into the range.
        "mel_fmin": null,         // minimum freq level for mel-spec. ~50 for male and ~95 for female voices. Tune for dataset!!
        "mel_fmax": null        // maximum freq level for mel-spec. Tune for dataset!!
    },

    "embedding_size": 256,    
    "text_cleaner": "english_cleaners",
    "epochs": 1000,
    
    "lr": 0.0001,
    "lr_decay": false,
    "warmup_steps": 4000,

    "batch_size": 6,
    "eval_batch_size":4,
    "r": 1,
    "wd": 0.000001,
    "checkpoint": true,
    "save_step": 5000,
    "print_step": 10,

    "run_eval": true,
    "data_path": "../../Data/LJSpeech-1.1/",  // can overwritten from command argument
    "meta_file_train": "metadata_train.csv",      // metafile for training dataloader
    "meta_file_val": "metadata_val.csv",    // metafile for validation dataloader
    "data_loader": "TTSDataset",      // dataloader, ["TTSDataset", "TTSDatasetCached", "TTSDatasetMemory"]
    "dataset": "ljspeech",     // one of TTS.dataset.preprocessors, only valid id dataloader == "TTSDataset", rest uses "tts_cache" by default.
    "min_seq_len": 0,
    "output_path": "../keep/",
    "num_loader_workers": 4
}
```

**Comment by xsr6064** [comment]
Thanks for the updated config file. Could you explain a little more about the parameter 'wd' in the config? I see it is used in updating the model parameters after the back-prop step (before the 'check_update' method), but don't quite get the purpose of the update. Is it just a numeric operation for testing if the gradients are valid? Sorry for keeping bugging you with simple questions, and thanks a lot.
